.github/workflows/pr.yml

```diff
@@ -100,6 +100,11 @@ jobs:
       run: npm run deploy:flutter
       working-directory: importer
 
+    - name: Replace version number in flutter/pubspec.yaml
+      run: |
+        sed -i.bk -r "version: s/[0-9]+\.[0-9]+\.[0-9]+/$NEW_VERSION/g" flutter/pubspec.yaml
```

Comment:
I think it should be `sed -i.bk -r "version: s/[0-9]+\.[0-9]+\.[0-9]+/version: $NEW_VERSION/g" flutter/pubspec.yaml`

Reply:
#122